content/zh-cn/blog/_posts/2018-07-12-resizing-persistent-volumes-using-kubernetes.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: blog
+title: '使用 Kubernetes 调整 PersistentVolume 的大小'
+date: 2018-07-12
+---
```

Comment:
Add a slug below line 4:
```
slug: resize-pv-using-k8s
```
since the current url is: `/zh-cn/blog/2018/07/12/使用-kubernetes-调整-persistentvolume-的大小/`

Reply:
Thank you for the review.